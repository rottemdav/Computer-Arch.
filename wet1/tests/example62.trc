16 8 20 3 global_history local_tables using_share_lsb
0x4680 N 0xc24
0xc3a584 N 0xd2c
0xc040d0 N 0x820283c
0xe560 N 0x562c
0xc98 N 0x681ea8
0x1370 T 0xd529d0
0x024 N 0xa20dee54
0xcc92aae0 T 0xaf7e0
0xe560 T 0x562c
0x120 T 0xac08
0x9e2954c N 0xee8
0xc040d0 N 0x820283c
0x4680 T 0xc24
0xcec N 0xedf910
0xcec N 0xedf910
0xc3a584 T 0xd2c
0x4680 T 0xc24
0x024 N 0xa20dee54
0x9f61c42c N 0xf2f03c
0x1370 T 0xd529d0
0x1146224 N 0x5a214c
0x70214 T 0xb171fc
0xc98 T 0x681ea8
0xdc8c T 0xf4a3159c
0x9f61c42c T 0xf2f03c
0x9e2954c N 0xee8
0x024 T 0xa20dee54
0xda0 T 0x9110
0x421ac08 T 0xcdfc
0x024 N 0xa20dee54
0xc040d0 T 0x820283c
0x54ede60 N 0x2ec
0xcc92aae0 N 0xaf7e0
0xcc92aae0 T 0xaf7e0
0xcec T 0xedf910
0x12b0 N 0x016b87b0
0xcec T 0xedf910
0xda0 T 0x9110
0xc3a584 T 0xd2c
0x1146224 N 0x5a214c
0x98c9cc8 T 0x17a5514
0x024 N 0xa20dee54
0xe09d2c T 0x778
0x9f61c42c N 0xf2f03c
0x421ac08 T 0xcdfc
0x9b65c7a4 N 0x54f00334
0x9e2954c T 0xee8
0x9b65c7a4 T 0x54f00334
0x4680 N 0xc24
0x54ede60 T 0x2ec
0x120 T 0xac08
0x421ac08 N 0xcdfc
0x9e2954c T 0xee8
0x9e2954c T 0xee8
0x1146224 N 0x5a214c
0x1146224 T 0x5a214c
0x7d6b8ec T 0x82ab294
0x421ac08 N 0xcdfc
0xc040d0 N 0x820283c
0xc5348c N 0x4a0
0xe560 N 0x562c
0xc040d0 T 0x820283c
0x54ede60 N 0x2ec
0xc5348c T 0x4a0
0xe560 N 0x562c
0x1370 N 0xd529d0
0x9e2954c N 0xee8
0x9e2954c T 0xee8
0xa6004a8 T 0x4142d504
0xcc92aae0 N 0xaf7e0
0xcec N 0xedf910
0xe09d2c T 0x778
0x7d6b8ec T 0x82ab294
0x1f4 N 0xb740
0x120 N 0xac08
0x1146224 N 0x5a214c
0x9e2954c T 0xee8
0x4680 N 0xc24
0xda0 N 0x9110
0x1146224 N 0x5a214c
0x9b65c7a4 N 0x54f00334
0xd10ec0 N 0x7d7b648
0x54ede60 T 0x2ec