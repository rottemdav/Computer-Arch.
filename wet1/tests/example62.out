0x4680 N 0x4684
0xc3a584 N 0xc3a588
0xc040d0 N 0xc040d4
0xe560 N 0xe564
0xc98 N 0xc9c
0x1370 N 0x1374
0x24 N 0x28
0xcc92aae0 N 0xcc92aae4
0xe560 N 0xe564
0x120 N 0x124
0x9e2954c N 0x9e29550
0xc040d0 T 0x820283c
0x4680 T 0xc24
0xcec N 0xcf0
0xcec T 0xedf910
0xc3a584 T 0xd2c
0x4680 T 0xc24
0x24 T 0xa20dee54
0x9f61c42c N 0x9f61c430
0x1370 T 0xd529d0
0x1146224 N 0x1146228
0x70214 N 0x70218
0xc98 T 0x681ea8
0xdc8c N 0xdc90
0x9f61c42c T 0xf2f03c
0x9e2954c N 0x9e29550
0x24 N 0x28
0xda0 N 0xda4
0x421ac08 N 0x421ac0c
0x24 T 0xa20dee54
0xc040d0 T 0x820283c
0x54ede60 N 0x54ede64
0xcc92aae0 N 0xcc92aae4
0xcc92aae0 T 0xaf7e0
0xcec N 0xcf0
0x12b0 N 0x12b4
0xcec T 0xedf910
0xda0 N 0xda4
0xc3a584 T 0xd2c
0x1146224 N 0x1146228
0x98c9cc8 N 0x98c9ccc
0x24 N 0x28
0xe09d2c N 0xe09d30
0x9f61c42c N 0x9f61c430
0x421ac08 N 0x421ac0c
0x9b65c7a4 N 0x9b65c7a8
0x9e2954c T 0xee8
0x9b65c7a4 T 0x54f00334
0x4680 T 0xc24
0x54ede60 N 0x54ede64
0x120 N 0x124
0x421ac08 T 0xcdfc
0x9e2954c T 0xee8
0x9e2954c T 0xee8
0x1146224 N 0x1146228
0x1146224 T 0x5a214c
0x7d6b8ec N 0x7d6b8f0
0x421ac08 T 0xcdfc
0xc040d0 T 0x820283c
0xc5348c N 0xc53490
0xe560 N 0xe564
0xc040d0 T 0x820283c
0x54ede60 N 0x54ede64
0xc5348c T 0x4a0
0xe560 N 0xe564
0x1370 N 0x1374
0x9e2954c N 0x9e29550
0x9e2954c T 0xee8
0xa6004a8 N 0xa6004ac
0xcc92aae0 N 0xcc92aae4
0xcec N 0xcf0
0xe09d2c N 0xe09d30
0x7d6b8ec N 0x7d6b8f0
0x1f4 N 0x1f8
0x120 N 0x124
0x1146224 T 0x5a214c
0x9e2954c T 0xee8
0x4680 T 0xc24
0xda0 N 0xda4
0x1146224 T 0x5a214c
0x9b65c7a4 N 0x9b65c7a8
0xd10ec0 N 0xd10ec4
0x54ede60 N 0x54ede64
flush_num: 32, br_num: 83, size: 9016b
